core/src/server/web_impl/WebServer.cpp

```diff
@@ -56,8 +56,6 @@ WebServer::StartService() {
     status = config.GetServerConfigWebPort(port);
 
     {
-        AppComponent components = AppComponent(std::stoi(port));
```

Comment:
Do not remove Appcomponent. It's used to initialize http server handler, provider.